File: cpp/02_variables.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string name = "ulas";
    int age = 20;
    double height = 1.75;
    bool isStudent = true;

    cout << name << " " << age << " " << height << " " << isStudent << endl;

    age = 21;
    cout << "new age: " << age << endl;

    return 0;
}

File: go/02_variables.go
package main

import "fmt"

func main() {
	var name string = "ulas"
	var age int = 20
	var height float64 = 1.75
	var isStudent bool = true

	fmt.Println(name, age, height, isStudent)

	// short declaration
	age2 := 21
	fmt.Println("new age:", age2)

	fmt.Printf("type of name: %T\n", name)
}
